src/methods.jl

```diff
@@ -1,5 +1,36 @@
 # helper function
 axes_min_length(idxs) = argmin([a isa Colon ? Inf : length(a) for a in idxs])
+axes_min_length(idxs::String) = axes_min_length(fits_indices(idxs))
+
+"""
+    fits_indices(string::String)
+
+Converts FITS based indices to Julian indices.
+
+# Examples
+```jldoctest
+julia> fits_indices("[1024:2048, :]")
+2-element Array{Any,1}:
+ Colon()
+ 1024:2048
+
+```
+"""
+function fits_indices(string::String)
+    str = replace(replace(replace(string, "[" => ""), "]" => ""), " " => "")
+    tokens = split(str, ',')
+
+    idxs = map(tokens) do token
+        t = split(token, ':')
+        if all(i -> i == "", t)
+            Colon()
+        else
+            low, high = parse.(Int, t)
+            low : high
+        end
```

Comment:
technically `"[1024:2048, :]"` is not a valid FITS index, it has to be `"[1024:2048, 1:2048]"`. I would change this in the example, but I think it's a good idea to keep this code.
```suggestion
        t = split(token, ':', keepempty=false)
        length(t) == 0 ? Colon() : parse(Int, t[1]):parse(Int, t[2])
```